notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1882 @@
+{
```

Comment:
<p>Could you clarify why you're doing this? Is this something we should also document in the public doc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>No need to add to our public doc, this is a feature from KFP. This is just to provide some quick code snippet to help the users. I have seen a lot of customers asking about that, so I put it here in case the customers have questions.</p>